cypress/e2e/API/EngageSphere.cy.js

```diff
@@ -0,0 +1,150 @@
+const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+
+describe('Teste de API - Filtragem de clientes por tamanho', () => {
+    const sizes = ['Small', 'Medium', 'Enterprise', 'Large Enterprise', 'Very Large Enterprise'];
+  
+    sizes.forEach((size) => {
+      it(`Deve retornar apenas clientes com o tamanho "${size}"`, () => {
+        cy.request('GET', `${CUSTOMERS_API_URL}?page=1&limit=50&size=${size}`)
+          .then((response) => {
```

Comment:
Que tal usar desestruturação de objetos? Veja abaixo.

```suggestion
          .then(({ status, body }) => {
```

> **O mesmo vale para os outros testes.**